ingestion/src/metadata/ingestion/source/database/redshift/utils.py

```diff
@@ -395,3 +398,16 @@ def _get_all_relation_info(self, connection, **kw):  # pylint: disable=unused-ar
         key = RelationKey(rel.relname, rel.schema, connection)
         relations[key] = rel
     return relations
+
+
+@reflection.cache
+def get_table_ddl(
+    self, connection, table_name, schema=None, **kw
+):  # pylint: disable=unused-argument
+    return get_table_ddl_wrapper(
+        self,
+        connection=connection,
+        query=None,
+        table_name=table_name,
+        schema=schema,
+    )
```

Comment:
We have duplicate code on all database utils, if we can have that in a common util for all dbs..

Reply:
i have shift this code block into the `sqlalchemy_utils.py` file.